module/applications/actor/group-sheet.mjs

```diff
@@ -1,433 +1,487 @@
-      secrets: this.actor.isOwner,
-      rollData: context.rollData,
-      relativeTo: this.actor
-    });
+  /** @override */
+  async _configureInventorySections(sections) {
+    sections.forEach(s => s.minWidth = 200);
+  }
 
-    // Summary tag
-    context.summary = this.#getSummary(stats);
+  /* -------------------------------------------- */
 
-    // Text labels
-    context.labels = {
-      currencies: Object.entries(CONFIG.DND5E.currencies).reduce((obj, [k, c]) => {
-        obj[k] = c.label;
-        return obj;
-      }, {})
+  /**
+   * Prepare rendering context for the biography tab.
+   * @param {ApplicationRenderContext} context  Context being prepared.
+   * @param {HandlebarsRenderOptions} options   Options which configure application rendering behavior.
+   * @returns {ApplicationRenderContext}
+   * @protected
+   */
+  async _prepareBiographyContext(context, options) {
+    if ( this.actor.limited ) return context;
+
+    const enrichmentOptions = {
+      secrets: this.actor.isOwner, relativeTo: this.actor, rollData: context.rollData
+    };
+    context.enriched = {
+      summary: await CONFIG.ux.TextEditor.enrichHTML(this.actor.system.description.summary, enrichmentOptions),
+      full: await CONFIG.ux.TextEditor.enrichHTML(this.actor.system.description.full, enrichmentOptions)
     };
+    if ( this.editingDescriptionTarget ) context.editingDescription = {
+      target: this.editingDescriptionTarget,
+      value: foundry.utils.getProperty(this.actor._source, this.editingDescriptionTarget)
+    };
+
     return context;
   }
 
   /* -------------------------------------------- */
 
   /**
-   * Prepare a localized summary of group membership.
-   * @param {{nMembers: number, nVehicles: number}} stats     The number of members in the group
-   * @returns {string}                                        The formatted summary string
+   * Prepare render context for player characters.
+   * @param {Actor5e} actor                    The player character actor.
+   * @param {object} context                   The render context.
+   * @param {HandlebarsRenderOptions} options  Options which configure application rendering behavior.
+   * @returns {Promise<void>}
+   * @protected
    */
-  #getSummary(stats) {
-    const formatter = game.i18n.getListFormatter({ style: "long", type: "conjunction" });
-    const rule = new Intl.PluralRules(game.i18n.lang);
-    const members = [];
-    if ( stats.nMembers ) {
-      members.push(`${stats.nMembers} ${game.i18n.localize(`DND5E.Group.Member.${rule.select(stats.nMembers)}`)}`);
-    }
-    if ( stats.nVehicles ) {
-      members.push(`${stats.nVehicles} ${game.i18n.localize(`DND5E.Group.Vehicle.${rule.select(stats.nVehicles)}`)}`);
-    }
-    if ( !members.length ) return game.i18n.localize("DND5E.GroupSummaryEmpty");
-    return game.i18n.format("DND5E.GroupSummary", {members: formatter.format(members)});
+  async _prepareCharacterContext(actor, context, options) {
```

Comment:
For the broad organization of the other new actor sheets, we've moved these more specific preparation methods into their own section beneath the "Rendering" section:

- Rendering (for part context preparation and other general application rendering methods)
- Actor Preparation Helpers (for preparation methods related to the actor itself)
- Item Preparation Helpers (for preparation methods related to individual items on the actor)

Reply:
Not 100% on the demarcation between these methods. I've re-arranged them. Let me know if it's what you had in mind.

Reply:
Yeah, that matches what I did on the other sheets.